ext/opentelemetry-ext-flask/tests/test_flask_integration.py

```diff
@@ -76,6 +76,35 @@ def assert_environ():
         self.client.get("/assert_environ")
         self.assertEqual(nonstring_keys, set())
 
+    def test_copy_current_request(self):
```

Comment:
This is something that needs to be fixed though, and if we can't find another way then I'm even fine with using `update_name`.